roles/lib_openshift/test/test_oc_csr_approve.py

```diff
@@ -16,6 +16,7 @@
 sys.path.insert(1, MODULE_PATH)
```

Comment:
Minimal changes to unit tests, just updating to use instance methods instead of module functions.